itou/archive/management/commands/notify_archive_jobseekers.py

```diff
@@ -18,7 +18,7 @@
 
 logger = logging.getLogger(__name__)
 
-INACTIVITY_PERIOD = datetime.timedelta(days=365)
+INACTIVITY_PERIOD = datetime.timedelta(days=365) * 2
```

Comment:
De ce que j'ai compris, on veux que les données soit supprimées à 24 mois, là ça serais 25 mois de ce que j'ai compris du script.
```suggestion
INACTIVITY_PERIOD = datetime.timedelta(days=365) * 2 - GRACE_PERIOD
```